Interfaces/WidgetInterface.php

```diff
@@ -429,6 +429,12 @@ public function getPage();
      * @return string
      */
     public function getPageId();
+    
+    /**
+     * 
+     * @return string
+     */
+    public function getPageAlias();
```

Comment:
Auch hier lieber keine Proxy-Methoden einbauen $widget->getPage()->getAlias() ist ausreichend. Ich würde eigentlich auch die getPageId() entfernen. Die wird scheinbar in Templates dafür verwendet, zu prüfen, ob zwei seiten gleich sind. Das müssen wir anders lösen: z.B. $widgetLink->getPage()->is($widget->getPage()). Ids vergleichen reicht nicht mehr aus, weil wir Seiten durch andere ersetzen können und die hätten eigene IDs. Ich würde der Page zwei Methoden dafür geben: $page->isExactly($other_page) vergleicht die UIDs und $page->is($other_page) liefert auch dann TRUE, wenn $other_page $page ersetzen soll.